ExtractData/PKHeX.ExtractData/Utils/ExtractUtils.cs

```diff
@@ -0,0 +1,382 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PKHeX.ExtractData
+{
+    class ExtractUtils
+    {
+        internal static ushort[] SearchMovesId(string[] SearchMoves)
+        {
+            string[] all_moves = Util.getMovesList("en");
+            ushort[] Id_SearchMoves = new ushort[SearchMoves.Length];
+
+            for (int i = 0; i < Id_SearchMoves.Length; i++)
+            {
+                string move_find = SearchMoves[i].Trim();
+                if (all_moves.Contains(move_find))
+                {
+                    Id_SearchMoves[i] = (ushort)all_moves.
+                            Select((move, index) => new { index = index, move = move }).
+                            Where(x => x.move == move_find).
+                            Select(x => x.index).
+                            First();
+                }
+                else
+                {
+                    object a = new object();
```

Comment:
Sorry, i forgot to delete that function. That was used to find some of the offset of the moves data, most of the code to find the offset was deleted because is not needed. This function is not used anymore.